### ISSUE TITLE
Update Fonts related to LegacyTabCell

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#21112]
_This task is part of a series of tasks to standardizing fonts to start using FXFontStyles._

**Background Context:**
Previously, we were using `DefaultDynamicFontHelper` and setting the text style, size and weight for each font, which sometimes did not match our design system. By using `FXFontStyles`, we can standardize our fonts and be more aligned with the design system.

**Task:**
Please update how we set fonts in the Firefox iOS Project related:

`LegacyTabCell`

**Acceptance Criteria:**

Replace usage of `DefaultDynamicFontHelper` with `FXFontStyles`

If there are discrepancies in terms of replacing the old fonts with the new standard, we will need to bring design in to approve the PR.

Please provide before and after screenshots of the UI.

**Reference:**

Usage example:
`FXFontStyles.Regular.headline.scaledFont()`

See this PR for example of standardizing fonts for Primary Button: 

[https://github.com/mozilla-mobile/firefox-ios/pull/18711/files](https://github.com/mozilla-mobile/firefox-ios/pull/18711/files|smart-link)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-9541)

**Comment by cyndichin** [comment]
thank you for being interested in this issue, assigned the ticket to you.

**Comment by cyndichin** [comment]
@yusuphjoluwasen wanted to follow up if you were still interested in working on this, thank you!

**Comment by yusuphjoluwasen** [comment]
@cyndichin Yes, I am interested in working on the issue. I will submit a pull request before the end of the week. Thanks

**Comment by cyndichin** [comment]
great, thank you!

**Comment by yusuphjoluwasen** [comment]
Hi @cyndichin,

I hope you don't mind, but could I trouble you for a screenshot of the legacy tab? I’ve been trying to find it in the app.
Many thanks

**Comment by data-sync-user** [comment]
➤ Cyndi Chin commented:

For QA, please test around the tab tray area, see screenshots in PR. This should be our legacy tab instead of the tab tray refactor though.

**Comment by data-sync-user** [comment]
➤ Diana Andreea Barladeanu commented:

Validated on v132 (45864), with iPhone 15 (17.5).
